src/Compilers/CSharp/Portable/Symbols/Source/SourceNamedTypeSymbol_Bases.cs

```diff
@@ -91,10 +91,12 @@ protected override void CheckBase(DiagnosticBag diagnostics)
                 return;
             }
 
+            Location baseLocation = null;
```

Comment:
>baseLocation [](start = 21, length = 12)

It looks like all code paths assign `baseLocation`, and always to the same value. Why not assign here?